library/src/main/java/org/xmtp/android/library/ApiClient.kt

```diff
@@ -77,16 +78,22 @@ data class GRPCApiClient(override val environment: XMTPEnvironment, val secure:
                 }
                 if (pagination?.startTime != null) {
                     it.endTimeNs = pagination.startTime.time * 1_000_000
-                    it.pagingInfoBuilder.direction =
```

Comment:
The move to lite mostly just got rid of our fancy builder methods which just makes the code a little more verbose. Hopefully with the move to more kotlin protobufs we can clean a lot of this builder code up.